docs/data-operations-manual/How-To-Guides/Adding/Configure-an-endpoint.md

```diff
@@ -107,6 +107,20 @@ Important fields:
 
 _I think to set a default value using another field in the resource, but uncertain how this is different to column. Need more info._
 
+## [pipeline/entity-organisation](https://github.com/digital-land/specification/blob/main/content/dataset/entity-organisation.md?plain=1)
+
+Used to set the entity range for organisations within the conservation-area collection. This is done to ensure that entities within a range are linked to a certain organisation.
```

Comment:
Nice explanation! Maybe just add on something like "in the case that we have lookup entries for the same entity from different organisations" to make it clearer that this helps prioritise data from the authoritative source.